ts/components/ui/ListItemInfoCopy.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from "react";
+import { useCallback } from "react";
+import {
+  View,
+  StyleSheet,
+  Pressable,
+  GestureResponderEvent,
+  Text
+} from "react-native";
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withSpring,
+  useDerivedValue,
+  interpolate,
+  Extrapolate,
+  interpolateColor
+} from "react-native-reanimated";
+import { Icon, IOIcons } from "../core/icons";
+import {
+  IOListItemStyles,
+  IOListItemVisualParams,
+  IOStyles
+} from "../core/variables/IOStyles";
+import { IOSpringValues, IOScaleValues } from "../core/variables/IOAnimations";
+import { LabelSmall } from "../core/typography/LabelSmall";
+import { IOColors, hexToRgba, useIOTheme } from "../core/variables/IOColors";
+import { WithTestID } from "../../types/WithTestID";
+import { useIOSelector } from "../../store/hooks";
+import { makeFontStyleObject } from "../core/fonts";
+import { isDesignSystemEnabledSelector } from "../../store/reducers/persistedPreferences";
+import { NewH6 } from "../core/typography/NewH6";
+import { Body } from "../core/typography/Body";
+
+export type ListItemInfoCopy = WithTestID<{
+  label: string;
```

Comment:
In some of our design we have this component without the label (for example, [here](https://www.figma.com/file/SuYjyjZlieQqRS8n1B6qbF/Flussi-App-ID-Pay?type=design&node-id=5004-31215&t=guqKj5LdCNrRo7w5-4)). What do you think about making `label` optional?